lib/jxl/dec_modular.cc

```diff
@@ -622,8 +636,34 @@ Status ModularFrameDecoder::FinalizeDecoding(PassesDecoderState* dec_state,
 
   auto& decoded = dec_state->decoded;
 
-  JXL_RETURN_IF_ERROR(
-      ModularImageToDecodedRect(gi, dec_state, pool, output, Rect(decoded)));
+  if (dec_state->render_pipeline) {
+    std::atomic<size_t> num_errors{0};
```

Comment:
[In the past](https://github.com/libjxl/libjxl/pull/860/files#diff-c7ffee7b87f9e62e559021b02af4ddbeae7d920d0ac7bde1a3b8322e4a9319adR45), I have used a bool for this. Is a size_t preferable? It does maybe seem a bit lighter than the bool notation-wise.

Reply:
Done.